Jenkinsfile

```diff
@@ -11,18 +11,18 @@ elifePipeline {
         {
             stage 'Build images', {
                 checkout scm
-                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.ci.yml build"
+                sh "IMAGE_TAG=${commit} COMPOSER_TYPE=no-dev docker-compose build"
             }
 
             stage 'Project tests', {
                 try {
-                    sh "chmod 777 build/ && IMAGE_TAG=${commit} docker-compose -f docker-compose.ci.yml run --rm ci ./project_tests.sh"
+                    sh "chmod 777 build/ && IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm ci ./project_tests.sh"
```

Comment:
isn't the double `-f` with both the `dev` and `ci` files necessary everywhere `docker-compose` is called in the build?